### DISCUSSION TITLE
Equipos de Hackaton?

### DISCUSSION THREAD

**Discussion by gjordanra** [Launch-X-Latam/MisionFrontEnd#559] [Team Posts]
Apenas estamos empezando la mision backend y no se que paso que es una locura de mensajes de equipos de Hackaton, muchos ni llegaremos hasta allá. Pero ya  que estamos en eso, me gustaria si hay algunos que quieran formar equipo y no piensen renunciar, lo ideal sería con otros que tengan situaciones similares, que estén trabajando y que tengan una zona horaria similar o de 1 hora más o 1 hora menos, por aquello de los tiempos y la comunicación, mi zona es GMT-5.

**Comment by Eduar2TC** [comment]
Tienes razón es una locura. 
No se si alguien dijo que habría Hackaton y me sorprendió ver mucho spam en mi email ver gente armando equipos prematuramente.

**Comment by DavidSantana872** [comment]
Me pasó lo mismo de la nada todos formando equipo para el hackaton me asuste machín

**Comment by gjordanra** [comment]
Uno de los requisitos es participar en el Hackaton al final, pero no he visto que nos pidan hacer los equipos y faltan 3 meses

**Comment by gjordanra** [comment]
si se que no podria hacer equipo con otros que tengan mucha diferencia horaria porque estoy trabajando

**Comment by gjordanra** [comment]
Bueno ya vi el video que me perdí, y el sherpa dijo claro, no les puedo decir aun nada, solo prepárense, no se si me perdí algo mas, pero si es solo eso aun no se sabe de cuánto son los grupos ni cómo deben ser

**Comment by AlexG0712** [comment]
Los equipos son de 5, yo le pregunté el la sesión pasada

**Comment by dansuca** [comment]
Hola chicos, …si alguien le falta integrante, escríbanme


Enviado desde Correo<https://go.microsoft.com/fwlink/?LinkId=550986> para Windows 10

De: Gustavo Jordán ***@***.***>
Enviado: sábado, 12 de noviembre de 2022 23:34
CC: ***@***.***>
Asunto: Re: [Launch-X-Latam/mision-backend] Equipos de Hackaton? (#96)


Bueno ya vi el video que me perdí, y el sherpa dijo claro, no les puedo decir aun nada, solo prepárense, no se si me perdí algo mas, pero si es solo eso aun no se sabe de cuánto son los grupos ni cómo deben ser

**Comment by Max80318** [comment]
De que trata el hackaton

**Comment by gjordanra** [comment]
> Los equipos son de 5, yo le pregunté el la sesión pasada

MMM bueno no vi esa parte

**Comment by gjordanra** [comment]
> De que trata el hackaton
> 
> El sáb., 12 nov. 2022 11:21 p. m., Daniela Suazo ***@***.***>
> escribió:
> …

aun no se sabe, solo se sabe que hay que hacer un proyecto para entregar en grupo

**Comment by Skuanquis** [comment]
Yo me apunto pero no se si aun se puede mi zona es GMT-4
